drivers/storage/class/disk/disk.inf

```diff
@@ -212,3 +212,20 @@ compaq_array.DeviceDesc = "Compaq SCSI дисковый массив"
 mylex_array.DeviceDesc = "Mylex RAID дисковое устройство"
 
 VHD_Generic_FriendlyName = "Microsoft виртуальный диск"
+
+[Strings.0a]
+MSFT="Microsoft"
```

Comment:
```suggestion
```

Reply:
Ok, same as the comment before. Avoid dupes.